vita3k/gui/src/live_area.cpp

```diff
@@ -1046,35 +1047,38 @@ void draw_live_area_screen(GuiState &gui, EmuEnvState &emuenv) {
+                ImGui::TextColored(GUI_COLOR_TEXT, "%s", fw_str);
             }
             ImGui::Spacing();
             ImGui::Separator();
             ImGui::Spacing();
-            ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Live Area Help");
+            ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%s", lang["live_area_help"].c_str());
             ImGui::Spacing();
-            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", "Browse in app list", "D-pad, Left Stick, Wheel in Up/Down or using Slider");
-            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", "Start App", "Click on Start or Press on Cross");
-            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", "Show/Hide Live Area during app run", "Press on PS");
-            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", "Exit Live Area", "Click on Esc or Press on Circle");
+            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", lang["browse_app"].c_str(), lang["browse_app_control"].c_str());
+            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", lang["start_app"].c_str(), lang["start_app_control"].c_str());
+            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", lang["show_hide"].c_str(), lang["show_hide_control"].c_str());
+            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", lang["exit_livearea"].c_str(), lang["exit_livearea_control"].c_str());
             ImGui::Spacing();
             ImGui::Separator();
             ImGui::Spacing();
-            ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Manual Help");
+            ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%s", lang["manual_help"].c_str());
             ImGui::Spacing();
-            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", "Browse page", "D-pad, Left Stick in Left/Right or Wheel in Up/Down or Click on </>");
-            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", "Hide/Show button", "Right Click");
-            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", "Zoom if available", "Double left click");
-            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", "Scroll in zoom", "Wheel Up/Down");
-            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", "Exit Manual", "Click on X or Press on PS");
+            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s </>", lang["browse_page"].c_str(), lang["browse_page_control"].c_str());
+            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", lang["hide_show"].c_str(), lang["hide_show_control"].c_str());
+            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", lang["zoom_available"].c_str(), lang["zoom_available_control"].c_str());
+            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", lang["scroll_zoom"].c_str(), lang["scroll_zoom_control"].c_str());
+            ImGui::TextColored(GUI_COLOR_TEXT, "%-16s    %-16s", lang["exit_manual"].c_str(), lang["exit_manual_control"].c_str());
```

Comment:
juste revert this file, live area part only need continue/start, i want keep enum for both